app/uk/gov/hmrc/eusubsidycompliancefrontend/services/SubsidyJourney.scala

```diff
@@ -39,6 +40,7 @@ case class SubsidyJourney(
       .fold(List.empty[Any])(identity)
       .map(_.cast[FormPage[_]])
 
+  def isAmend(): Boolean = existingTransactionId.nonEmpty
```

Comment:
can we rename this function to `doesTransactionIdExist` or any other name. isAmend doesn't seems to be appropriate

Reply:
I wanted to make it consistent with the Ammendment controller. It can be inferred by the existence of a transactionID

Reply:
it's not the end of the world , we can rename it later 👍